.gitignore

```diff
@@ -32,3 +32,6 @@ node_modules/
 build/lib
 src/SubOctpInterface.cpp
 src/SubOctpInterface.js
+build_dir/
+src/build_dir/
```

Comment:
Any of this that is used for Meson's working build directory are not needed -- Meson already creates a `build_dir/.gitignore` file containing `*` for you, on the principle that it is always correct to ignore an out of tree build directory, and in most cases the project doesn't know which directory the user selects (but of course Meson does know).

Reply:
Thanks, I didn't know that

Reply:
I am one of the Meson developers. :)

It's a fun easter egg to tell people about.